client/web/compose/src/components/PageBlocks/RecordRevisionsBase.vue

```diff
@@ -26,65 +28,86 @@
-                  <b-th>{{ $t('changes.columns.old-value.label') }}</b-th>
-                  <b-th>{{ $t('changes.columns.new-value.label') }}</b-th>
-                </b-tr>
-              </b-thead>
-              <b-tbody
-                v-for="(change) in row.item.changes"
-                :key="change.key"
+              <b-table-simple
+                class="bg-light"
               >
-                <b-tr>
-                  <b-td :rowspan="Math.max(change.new ? change.new.length : 0, change.old ? change.old.length : 0)">
-                    {{ change.key }}
-                  </b-td>
-                  <b-td>{{ change.old ? change.old[0] : '-' }}</b-td>
-                  <b-td>{{ change.new ? change.new[0] : '-' }}</b-td>
-                </b-tr>
-                <b-tr
-                  v-for="index in Math.max(change.new ? change.new.length - 1 : 0, change.old ? change.old.length - 1 : 0)"
-                  :key="change.key + index"
+                <b-thead>
+                  <b-tr>
+                    <b-th>{{ $t('changes.columns.field.label') }}</b-th>
+                    <b-th>{{ $t('changes.columns.old-value.label') }}</b-th>
+                    <b-th>{{ $t('changes.columns.new-value.label') }}</b-th>
+                  </b-tr>
+                </b-thead>
+                <b-tbody
+                  v-for="(change) in row.item.changes"
+                  :key="change.key"
                 >
-                  <b-td>{{ change.old && change.old.length > index ? change.old[index]: '-' }}</b-td>
-                  <b-td>{{ change.new && change.new.length > index ? change.new[index]: '-' }}</b-td>
-                </b-tr>
-              </b-tbody>
-            </b-table-simple>
-          </div>
-        </template>
-      </b-table-lite>
+                  <b-tr>
+                    <b-td :rowspan="Math.max(change.new ? change.new.length : 0, change.old ? change.old.length : 0)">
+                      {{ change.key }}
+                    </b-td>
+                    <b-td>{{ change.old ? change.old[0] : '-' }}</b-td>
+                    <b-td>{{ change.new ? change.new[0] : '-' }}</b-td>
+                  </b-tr>
+                  <b-tr
+                    v-for="index in Math.max(change.new ? change.new.length - 1 : 0, change.old ? change.old.length - 1 : 0)"
+                    :key="change.key + index"
+                  >
+                    <b-td>{{ change.old && change.old.length > index ? change.old[index]: '-' }}</b-td>
+                    <b-td>{{ change.new && change.new.length > index ? change.new[index]: '-' }}</b-td>
+                  </b-tr>
+                </b-tbody>
+              </b-table-simple>
+            </div>
+          </template>
+        </b-table-lite>
+
+        <div
+          v-if="!revisions.length"
+          class="position-absolute text-center mt-5 d-print-none"
+          style="left: 0; right: 0;"
+        >
+          <b-spinner
+            v-if="noRecord || processing"
```

Comment:
This is a duplicate of the spinner on line 18, I would prefer if you use that one and just remove this one to avoid edge cases